lib/vdsm/storage/lvmconf.py

```diff
@@ -184,3 +189,38 @@ def save(self):
     def close(self):
         log.debug("Closing LVM configuration %s", self.path)
         self.aug.close()
+
+
+def configured_value(section, option):
+    """
+    Return value configured for the option, taken into account all config
+    files or default configurations.
```

Comment:
I think "and" is more precise, because lvm must consider both config files and the defaults.

Reply:
done